docs/Using-Fleet/CIS-Benchmarks.md

```diff
@@ -32,10 +32,14 @@ All CIS policies are stored under our restricted licensed folder `ee/cis/`.
 
 How to import them to Fleet:
 ```sh
-# Download policy queries from Fleet's repository (e.g. for macOS 13)
+# Download policy queries from Fleet's repository 
+# macOS 13
 wget https://raw.githubusercontent.com/fleetdm/fleet/main/ee/cis/macos-13/cis-policy-queries.yml
 
-# Apply the downloaded policies to Fleet.
+# Windows 10 (note the same file name. Rename as needed.)
```

Comment:
Couple questions about usage to consider here:
+ Would the contents of these files need to be combined into one file (potentially with the rest of your policies) before running `fleetctl apply`? (IIRC when I was learning about the CLI and used a config file to update queries, its contents overrode all of our existing queries— but we don't seem to have docs for policy config files, so I'm not sure if this works the same way.)
+ Is there an alternative to using `wget` for this that we could show instead? it's not installed by default on macOS, so if there's an alternative path that doesn't involve having `wget` as a prerequisite, it'll be more approachable.

Reply:
Good questions.

Answer 1 : @lucasmrod can you confirm that `fleetctl apply` will override any existing policies?
If so, than @rachaelshaw you are right and the yml files will have to be merged. 
otherwise we're good to go separately.

Answer 2 : All our files are open to the public for viewing. We can just put the path and let the customer download it however they want:
https://github.com/fleetdm/fleet/blob/main/ee/cis/macos-13/cis-policy-queries.yml
But `wget` looks good to me.

Reply:
> Would the contents of these files need to be combined into one file (potentially with the rest of your policies) before running fleetctl apply? (IIRC when I was learning about the CLI and used a config file to update queries, its contents overrode all of our existing queries— but we don't seem to have docs for policy config files, so I'm not sure if this works the same way.)

No need to combine them, you can do:
```sh
fleetctl apply --context dogfood -f ee/cis/macos-13/cis-policy-queries.yml --policies-team "📊 CIS Benchmarks"
fleetctl apply --context dogfood -f ee/cis/win-10/cis-policy-queries.yml --policies-team "📊 CIS Benchmarks"
```
For policies it will apply the provided policies one by one. If a policy with the same name already exists then it will update its properties.

Reply:
Thanks @lucasmrod !
@rachaelshaw This makes things safe. Since the code is open source, the customers can chose how they download the files. We give them a way with wget.

TMWYT.